modules/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,179 @@
+. Leave *Template* unchanged, with *Blank* selected.
+. Set *Operating System* to *Red Hat Enterprise Linux CoreOS*.
+. Leave *Optimized for* unchanged, with *Desktop* selected.
+. For *Name*, specify the name of the virtual machine. For example, `vmname`.
+. For *Instance Images*, click btn:[Attach].
+. In the *Attach Virtual Disks* window that opens, find the disk you specified in the playbook. For example, `rhcos_44-81_img-diskname`.
+. Click the radio button for this disk.
+. Select the *OS* checkbox for this disk. This makes the disk bootable.
+. Click btn:[OK] to save and close the *Attach Virtual Disks* window.
+
+.Continue configuring the virtual machine and create it
+
+. In the *New Virtual Machine* -> *General* window, for each NIC under *Instantiate VM network interfaces by picking a vNIC profile*, select a vNIC profile.
+. Go to the *New Virtual Machine* -> *General* window.
++
+[IMPORTANT]
+====
+In the following steps, if you increase the values of *Memory* or *Total Virtual CPU* beyond the amounts shown, multiply these new values by the number nodes the installation program creates. By default, the installation program creates seven nodes. Verify that your {rh-virtualization} cluster has the memory and vCPU resources to meet these new values. For example, increasing *Total Virtual CPUs* from *4* to *6* means that your cluster requires 42 available vCPUs instead of 28. These vCPUs must be available exclusively for the cluster, and not required for non-cluster computing.
+====
++
+. Set *Memory* to *16384 MB* or more. This value is equivalent to 16 GiB. Adjust this value according to the workload you expect on the compute machines.
+. Due to link:https://bugzilla.redhat.com/show_bug.cgi?id=1821215[bz#1821215], set *Physical Memory Guaranteed* to *8192 MB*. This value is equivalent to 8 GiB.
+. Set *Total Virtual CPUs* to *4* or more.
+. Adjust *Cores per Virtual Socket* to *4* or more so it matches the *Total Virtual CPUs* setting.
+. Press btn:[OK] to save and close the *New Virtual Machine*.
++
+The new virtual machine appears in the *Compute* -> *Virtual Machines* window. Because the virtual machine is not starting, it should appear quickly.
+
+.Create a virtual machine template
+
+. Select the new virtual machine.
+. In the upper-right corner of the Administration Portal window, click the *More actions*, icon:ellipsis-v[], menu and select *Make Template*.
+. In the *New Template* window, specify a value for the *Name*. For example, `vm-tmpltname`.
+. Verify that *Target* and the other parameter values are correct and reflect your previous choices.
+. Verify that *Seal Template (Linux only)* is *not* selected.
+. Click btn:[OK].
+. Go to the *Compute* -> *Templates* page and wait for the template to be created.
+
+.Export some environment variables
+
+. Determine the values you specified for *Memory* and *Total Virtual CPUs* in the virtual machine settings.
```

Comment:
Is this the original value or the cumulative value? Also, it would make more sense to tell the user to note the values that they need when they're setting them.